.github/workflows/githubci.yml

```diff
@@ -9,19 +9,20 @@ jobs:
     
     steps:
     - uses: actions/checkout@v1
+    - uses: actions/setup-python@v1
+      with:
+        python-version: '3.x' # Version range or exact version of a Python version to use, using semvers version range syntax.
+        architecture: 'x64' # optional x64 or x86. Defaults to x64 if not specified
     - name: pre-install
       run: |
         source install.sh
-    - name: install libraries
-      run: |
-        arduino --install-library "Adafruit BusIO"
     - name: test platform
       run: |
-        build_platform esp32
-        build_platform uno
-        build_platform leonardo
-        build_platform zero
-        build_platform esp8266
-        build_platform m4
-        build_platform nrf52840
+        ./build.sh build_platform esp32
+        ./build.sh build_platform uno
+        ./build.sh build_platform leonardo
+        ./build.sh build_platform zero
+        ./build.sh build_platform esp8266
+        ./build.sh build_platform m4
+        ./build.sh build_platform nrf52840
```

Comment:
Made separate file for build because github was unable tu run it from the sourced one